### DISCUSSION TITLE
Solara apps work on mobile also a question

### DISCUSSION THREAD

**Discussion by saisaigraph** [widgetti/solara#160] [General]
Solara apps work on mobile also a question.
Thanks
Sai

**Comment by maartenbreddels** [comment]
Hi,

I think you mean "Does solara work on mobile?"
Yes it does! Did you try? Did something not work?

Regards,

Maarten

**Reply by saisaigraph** [reply]
Thanks Marteen Solaro looks great. Streamlit was my favorite but Solaro seems to be better.
